SPECS/cairo/cairo.spec

```diff
@@ -118,26 +123,62 @@ rm -rf %{buildroot}%{_datadir}/gtk-doc
 %{_libdir}/libcairo-script-interpreter.so.*
 
 %files devel
-%defattr(-,root,root)
-%dir %{_includedir}/%{name}
-%{_includedir}/%{name}/*
-%{_libdir}/*.so
-%{_libdir}/pkgconfig/*.pc
-
+%dir %{_includedir}/cairo/
+%{_includedir}/cairo/cairo-deprecated.h
+%{_includedir}/cairo/cairo-features.h
+%{_includedir}/cairo/cairo-ft.h
+%{_includedir}/cairo/cairo.h
+%{_includedir}/cairo/cairo-pdf.h
+%{_includedir}/cairo/cairo-ps.h
+%{_includedir}/cairo/cairo-script-interpreter.h
+%{_includedir}/cairo/cairo-svg.h
+%{_includedir}/cairo/cairo-tee.h
+%{_includedir}/cairo/cairo-version.h
+%{_includedir}/cairo/cairo-xlib-xrender.h
+%{_includedir}/cairo/cairo-xlib.h
+%{_includedir}/cairo/cairo-script.h
+%{_includedir}/cairo/cairo-xcb.h
+%{_libdir}/libcairo.so
+%{_libdir}/libcairo-script-interpreter.so
+%{_libdir}/pkgconfig/cairo-fc.pc
+%{_libdir}/pkgconfig/cairo-ft.pc
+%{_libdir}/pkgconfig/cairo.pc
+%{_libdir}/pkgconfig/cairo-pdf.pc
+%{_libdir}/pkgconfig/cairo-png.pc
+%{_libdir}/pkgconfig/cairo-ps.pc
+%{_libdir}/pkgconfig/cairo-script-interpreter.pc
+%{_libdir}/pkgconfig/cairo-svg.pc
+%{_libdir}/pkgconfig/cairo-tee.pc
+%{_libdir}/pkgconfig/cairo-xlib.pc
+%{_libdir}/pkgconfig/cairo-xlib-xrender.pc
+%{_libdir}/pkgconfig/cairo-script.pc
+%{_libdir}/pkgconfig/cairo-xcb-shm.pc
+%{_libdir}/pkgconfig/cairo-xcb.pc
+ 
 %files gobject
-%{_libdir}/libcairo-gobject.so.*
-
+%{_libdir}/libcairo-gobject.so.2*
+ 
 %files gobject-devel
-%{_includedir}/%{name}/cairo-gobject.h
+%{_includedir}/cairo/cairo-gobject.h
 %{_libdir}/libcairo-gobject.so
 %{_libdir}/pkgconfig/cairo-gobject.pc
-
+ 
 %files tools
 %license util/cairo-trace/COPYING util/cairo-trace/COPYING-GPL-3
 %{_bindir}/cairo-trace
-%{_libdir}/%{name}/
-
+%{_libdir}/cairo/
+ 
 %changelog
+* Mon Jan 29 2024 Sean Dougherty <sdougherty@microsoft.com> - 1.18.0-1
+- Upgrading to 1.18.0 for Mariner 3.0
```

Comment:
Suggestion (assuming you used other distro's specs): to give credit where it's due, you could also add a comment like `Used Fedora X spec (license: MIT) for guidance`.